README.md

```diff
@@ -161,6 +143,39 @@ docker container while running on the cluster. Here's the output:
 ...
 ```
 
+## Leveraging implicits
+
+In order to save some keystrokes, you can use our `implicit` operators:
+```scala
+import com.spotify.hype.magic._
+```
+
+Now you can set up an `implicit` `Submitter` value.
+```scala
+implicit val submitter = GkeSubmitter("gcp-project-id", "gce-zone-id", "gke-cluster-id", "gs://my-staging-bucket")
+```
+
+The environment value can also be declared `implicit`,
+but this is not required as it can explicitly be referenced when submitting functions.
+
+```scala
+implicit val env = RunEnvironment().withSecret("gcp-key", "/etc/gcloud")
+```
+
+Finally, use the `#!` (hashbang) operator to execute an `HFn[T]` in a given environment. It will
+use the `Submitter` and `RunEnvironment` which should be in scope.
+
+```scala
+val result = example("hello") #!
```

Comment:
it's using the example defined above. 
Perhaps it would make more sense to place `Leveraging implicits` before `Full example`?